debian/control

```diff
@@ -0,0 +1,98 @@
+Source: p4c
+Section: devel
+Priority: optional
+Maintainer: Thomas Dreibholz <dreibh@simula.no>
+Build-Depends: bison,
+               clang,
+               cmake,
+               debhelper (>= 12),
+               dh-python,
+               flex,
+               g++ (>= 9),
+               iproute2,
+               libboost-dev,
+               libboost-graph-dev,
+               libboost-iostreams-dev,
+               libelf-dev,
+               libfl-dev,
+               libgc-dev,
+               libgmp-dev,
+               libgrpc++-dev,
+               libgrpc-dev,
+               libprotobuf-dev,
+               libprotoc-dev,
+               libthrift-0.13.0,
+               libthrift-dev,
+               llvm,
+               net-tools,
+               p4lang-bmv2,
+               pkg-config,
+               protobuf-compiler,
+               protobuf-compiler-grpc,
+               python2,
+               python3,
+               python3-dev,
+               python3-grpcio,
+               python3-ipaddr,
+               python3-ply,
+               python3-protobuf,
+               python3-pyroute2,
+               python3-scapy,
+               python3-setuptools,
+               python3-thrift,
+               tcpdump,
+               thrift-compiler
+Standards-Version: 4.5.1.0
+Rules-Requires-Root: no
+Homepage: https://github.com/simula/p4c
+Vcs-Git: https://github.com/simula/p4c
+Vcs-Browser: https://github.com/simula/p4c/tree/p4-packaging
+
+Package: p4lang-p4c
+Architecture: amd64 arm64
+Depends: clang,
+         cpp,
+         iproute2,
+         libboost-dev,
+         libboost-graph-dev,
+         libboost-iostreams-dev,
+         libelf-dev,
+         libfl-dev,
+         libgc-dev,
+         libgmp-dev,
+         libgrpc++-dev,
+         libgrpc-dev,
+         libprotobuf-dev,
+         libprotoc-dev,
+         libthrift-0.13.0,
+         libthrift-dev,
+         llvm,
+         net-tools,
+         p4lang-bmv2,
+         pkg-config,
+         protobuf-compiler,
+         protobuf-compiler-grpc,
+         python3,
+         python3-dev,
+         python3-grpcio,
+         python3-ipaddr,
+         python3-ply,
+         python3-protobuf,
+         python3-pyroute2,
+         python3-scapy,
+         python3-setuptools,
+         python3-thrift,
+         tcpdump,
+         thrift-compiler,
+         ${misc:Depends},
+         ${python3:Depends},
+         ${python:Depends},
+         ${shlibs:Depends}
+Description: p4c p4lang project compiler
+ p4c is a new, alpha-quality reference compiler for the P4 programming language.
```

Comment:
Probably should use the updated description. p4c is not alpha-quality anymore.

Reply:
Yes, the first few sentences of this repo's top level README.md file were changed about a year ago to reflect its status better, and could be used as a starting point for the descriptive text here.

Reply:
Fixed.